sketch/Explorer_ESP_WiFi-localization_v1.ino

```diff
@@ -0,0 +1,158 @@
+#include <TheThingsNetwork.h>
+
+#define HEX_CHAR_TO_NIBBLE(c) ((c >= 'a') ? (c - 'a' + 0x0A) : (c - '0'))
+#define HEX_PAIR_TO_BYTE(h, l) ((HEX_CHAR_TO_NIBBLE(h) << 4) + HEX_CHAR_TO_NIBBLE(l))
+
+// Parameters for the esp stream/buffer
+#define ESP8266_DEFAULT_WAIT 5000
+#define ESP8266_BUFFER_SIZE 128
+#define ESP8266_SERIAL_SPEED 9600
+#define ACCESS_POINTS 3
+#define WIFI_BSSID_SIZE 6
+#define WAIT_TO_SEND 300000
+
+// Set your AppEUI and AppKey
+const char *appEui = "0000000000000000";
+const char *appKey = "00000000000000000000000000000000";
+
+// Replace REPLACE_ME with TTN_FP_EU868 or TTN_FP_US915
+#define FREQPLAN TTN_FP_EU868
+
+// Define the used serials
+#define DEBUGSERIAL SerialUSB
+#define LORASERIAL Serial2
+#define ESPSERIAL Serial
```

Comment:
Yes these are three exceptions, because they do not refer to a value but to an instance, they should be named as variables